src/django/api/models/state.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+from django.contrib.gis.db import models as gis_models
+
+__all__ = ["State"]
+
+
+class State(models.Model):
+    id = models.CharField(max_length=2, primary_key=True, unique=True)
+    name = models.CharField(max_length=127)
+    boundary = gis_models.PolygonField(geography=True)
```

Comment:
Since "Boundary" is an entity, with "shape" as the polygon field name, let's reuse that here for consistency.

```suggestion
    shape = gis_models.PolygonField(geography=True)
```

Reply:
Also, let's use a MultiPolygonField since many states have disconnected polygons in their shape (e.g. Michigan)